app/views/layouts/application.html.erb

```diff
@@ -16,6 +16,12 @@
     <link rel="apple-touch-icon-precomposed" href="/images/pl-255.png">
     <link rel="apple-touch-icon-precomposed apple-touch-icon" href="/images/pl-255.png">
     <link rel="shortcut icon" href="/images/pl-255.png" />
+    <script type="text/javascript" async
+      src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-MML-AM_CHTML">
+    </script>
+    <script type="text/x-mathjax-config">
+      MathJax.Hub.Config({tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]}});
```

Comment:
I think including this in every page is ok as it is needed in wikis, research notes and questions. Just need to test the browser compatabilites